source/contributing.md

```diff
@@ -23,14 +23,14 @@ git remote rename origin mainline
 
 6.  Add your remote repository in order to track it locally:
 ```
-git remote add username git@github.com:campa-consortium/particle-accelerator-lattice-standard.git
+git remote add username git@github.com:campa-consortium/pals.git
```

Comment:
Actually, besides your fix, shouldn't this be something like
```suggestion
git remote add username git@github.com:username/pals.git
```
assuming the user forked the repository? 

Otherwise doesn't this simply duplicate the `origin` remote?